docs/using-lagoon-the-basics/local-development-environments.md

```diff
@@ -22,8 +24,14 @@ brew tap pygmystack/pygmy && brew install pygmy
 
 For detailed usage or installation info on pygmy, see its [documentation](https://pygmy.readthedocs.io/en/master/).
 
-As announced in our [blog post](https://www.amazee.io/blog/post/announcing-lando-integration-for-lagoon), Lagoon is now also compatible with Lando! For more information, please see the documentation at [https://docs.lando.dev/config/lagoon.html](https://docs.lando.dev/config/lagoon.html) to get yourself up and running.
+### Lando
+
+Lagoon is now also compatible with Lando! For more information, please see the documentation at [https://docs.lando.dev/config/lagoon.html](https://docs.lando.dev/config/lagoon.html) to get yourself up and running.
```

Comment:
I wonder, given how long we've been compatible with Lando, whether it makes sense to change the phrasing from "is now also compatible" to something more appropriate for the long relationship we have with it?

Reply:
Updated!